ecs-init/Godeps/Godeps.json

```diff
@@ -15,8 +126,78 @@
 			"Rev": "c9ad0ce23f68428421adfc6ced9e6123f54788a5"
 		},
 		{
+			"ImportPath": "github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus",
```

Comment:
As per our offline sync: We should *not* merge this with the packages under `external`.  We can upgrade to Go 1.7, so if it's necessary to do that we should get that done first.